src/sentry/replays/usecases/replay_counts.py

```diff
@@ -56,6 +56,13 @@ def _get_replay_id_mappings(
         # if we want to validate list of replay_ids existence
         return {v: [v] for v in value}
 
+    # Any number of issues can be specified however we're only going to return at
+    # most 25 of them. Cap the lists length to 25. The other ids are silently
+    # dropped.
+    #
+    # NOTE: Should this be an error?
```

Comment:
Could we keep tabs on this through metrics? if we see a jump in calls with over 25 we can get eyes on it

Reply:
Definitely silently ignoring might be a problem in the future to figure out what's going on

Reply:
Adding a logger.warning which will appear in Sentry.